modules/web/js/ballerina/views/ballerina-file-editor.jsx

```diff
@@ -78,7 +78,9 @@ class BallerinaFileEditor extends React.Component {
             if (originEvtType === CHANGE_EVT_TYPES.TREE_MODIFIED) {
                 // Change was done from design view
                 // do an immediate update to reflect tree changes
-                this.forceUpdate();
+                if (this.ballerinaFileEditorWrapper) {
```

Comment:
Why do we need a ref check?

Reply:
just wanted to know the issue we are trying to solve here

Reply:
They have to be removed. Will be removing them in the next PR.

Reply:
Fixed with https://github.com/ballerinalang/composer/pull/3086/commits/5c737891cc1aa309dd89713ba98a8568966ee51f